### ISSUE TITLE
Double check that `revdep_install()` worked successfully

### ISSUE THREAD

**Issue by lionel-** [r-lib/revdepcheck#239]
I had strange revdep failures because the installation of the DEV version silently failed. The `new` folder for the tested package was empty, and the revdeps were then checked against an old version installed in the system library.

Checking that the package was correctly installed in old and new with the expected versions would help catch these problems early and would also increase our confidence that `remotes` etc are working properly during the revdep checks.

**Comment by lionel-** [comment]
Would it make sense to copy the package without the `revdep` folder somewhere inside `revdep` before installing?

Adding broken package and relaunching checks when there are hundreds of results in `checks.noindex` is very painful at the moment.

**Comment by gaborcsardi** [comment]
Yeah, that would be a good workaround for now, until pkgbuild or base R will get better at ignoring files.
